from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_setting_camera(object):
    def setupUi(self, setting_camera):
        setting_camera.setObjectName("setting_camera")
        setting_camera.resize(248, 350)
        self.label = QtWidgets.QLabel(setting_camera)
        self.label.setGeometry(QtCore.QRect(20, 30, 47, 13))
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(setting_camera)
        self.label_2.setGeometry(QtCore.QRect(20, 80, 47, 13))
        self.label_2.setObjectName("label_2")
        self.label_3 = QtWidgets.QLabel(setting_camera)
        self.label_3.setGeometry(QtCore.QRect(20, 120, 61, 16))
        self.label_3.setObjectName("label_3")
        self.label_4 = QtWidgets.QLabel(setting_camera)
        self.label_4.setGeometry(QtCore.QRect(20, 180, 47, 13))
        self.label_4.setObjectName("label_4")
        self.label_5 = QtWidgets.QLabel(setting_camera)
        self.label_5.setGeometry(QtCore.QRect(20, 220, 47, 16))
        self.label_5.setObjectName("label_5")
        self.label_6 = QtWidgets.QLabel(setting_camera)
        self.label_6.setGeometry(QtCore.QRect(20, 260, 47, 31))
        self.label_6.setObjectName("label_6")
        self.comboBox_name = QtWidgets.QComboBox(setting_camera)
        self.comboBox_name.setGeometry(QtCore.QRect(110, 20, 131, 22))
        self.comboBox_name.setObjectName("comboBox_name")
        self.comboBox_name.addItem("")
        self.comboBox_name.addItem("")
        self.comboBox_name.addItem("")
        self.comboBox_name.addItem("")
        self.comboBox_name.addItem("")
        self.comboBox_name.addItem("")
        self.comboBox_name.addItem("")
        self.comboBox_name.addItem("")
        self.comboBox_name.addItem("")
        self.comboBox_name.addItem("")
        self.comboBox_id = QtWidgets.QComboBox(setting_camera)
        self.comboBox_id.setGeometry(QtCore.QRect(110, 70, 131, 22))
        self.comboBox_id.setObjectName("comboBox_id")
        self.comboBox_id.addItem("")
        self.comboBox_id.addItem("")
        self.comboBox_id.addItem("")
        self.comboBox_id.addItem("")
        self.comboBox_id.addItem("")
        self.comboBox_id.addItem("")
        self.comboBox_id.addItem("")
        self.comboBox_id.addItem("")
        self.comboBox_id.addItem("")
        self.comboBox_id.addItem("")
        self.comboBox_conn = QtWidgets.QComboBox(setting_camera)
        self.comboBox_conn.setEnabled(True)
        self.comboBox_conn.setGeometry(QtCore.QRect(110, 120, 131, 22))
        self.comboBox_conn.setObjectName("comboBox_conn")
        self.comboBox_conn.addItem("")
        self.comboBox_conn.addItem("")
        self.comboBox_Brand = QtWidgets.QComboBox(setting_camera)
        self.comboBox_Brand.setGeometry(QtCore.QRect(110, 180, 131, 22))
        self.comboBox_Brand.setObjectName("comboBox_Brand")
        self.comboBox_Brand.addItem("")
        self.comboBox_Brand.addItem("")
        self.comboBox_Brand.addItem("")
        self.comboBox_Brand.addItem("")
        self.comboBox_Brand.addItem("")
        self.comboBox_Brand.addItem("")
        self.comboBox_Brand.addItem("")
        self.comboBox_Brand.addItem("")
        self.comboBox_Flip = QtWidgets.QComboBox(setting_camera)
        self.comboBox_Flip.setGeometry(QtCore.QRect(110, 220, 131, 22))
        self.comboBox_Flip.setObjectName("comboBox_Flip")
        self.comboBox_Flip.addItem("")
        self.comboBox_Flip.addItem("")
        self.comboBox_Flip.addItem("")
        self.comboBox_Flip.addItem("")
        self.comboBox_Rotate = QtWidgets.QComboBox(setting_camera)
        self.comboBox_Rotate.setGeometry(QtCore.QRect(110, 260, 131, 22))
        self.comboBox_Rotate.setObjectName("comboBox_Rotate")
        self.comboBox_Rotate.addItem("")
        self.comboBox_Rotate.addItem("")
        self.comboBox_Rotate.addItem("")
        self.comboBox_Rotate.addItem("")
        self.pushButton_ok = QtWidgets.QPushButton(setting_camera)
        self.pushButton_ok.setGeometry(QtCore.QRect(30, 310, 75, 23))
        self.pushButton_ok.setObjectName("pushButton_ok")
        self.pushButton_thoat = QtWidgets.QPushButton(setting_camera)
        self.pushButton_thoat.setGeometry(QtCore.QRect(140, 310, 75, 23))
        self.pushButton_thoat.setObjectName("pushButton_thoat")

        self.retranslateUi(setting_camera)
        QtCore.QMetaObject.connectSlotsByName(setting_camera)

    def retranslateUi(self, setting_camera):
        _translate = QtCore.QCoreApplication.translate
        setting_camera.setWindowTitle(_translate("setting_camera", "Camera setting"))
        self.label.setText(_translate("setting_camera", "Name:"))
        self.label_2.setText(_translate("setting_camera", "ID:"))
        self.label_3.setText(_translate("setting_camera", "Connection:"))
        self.label_4.setText(_translate("setting_camera", "Brand:"))
        self.label_5.setText(_translate("setting_camera", "Flip:"))
        self.label_6.setText(_translate("setting_camera", "Rotate:"))
        self.comboBox_name.setItemText(0, _translate("setting_camera", "Camera 0"))
        self.comboBox_name.setItemText(1, _translate("setting_camera", "Camera 1"))
        self.comboBox_name.setItemText(2, _translate("setting_camera", "Camera 2"))
        self.comboBox_name.setItemText(3, _translate("setting_camera", "Camera 3"))
        self.comboBox_name.setItemText(4, _translate("setting_camera", "Camera 4"))
        self.comboBox_name.setItemText(5, _translate("setting_camera", "Camera 5"))
        self.comboBox_name.setItemText(6, _translate("setting_camera", "Camera 6"))
        self.comboBox_name.setItemText(7, _translate("setting_camera", "Camera 7"))
        self.comboBox_name.setItemText(8, _translate("setting_camera", "Camera 8"))
        self.comboBox_name.setItemText(9, _translate("setting_camera", "Camera 9"))
        self.comboBox_id.setItemText(0, _translate("setting_camera", "0"))
        self.comboBox_id.setItemText(1, _translate("setting_camera", "1"))
        self.comboBox_id.setItemText(2, _translate("setting_camera", "2"))
        self.comboBox_id.setItemText(3, _translate("setting_camera", "3"))
        self.comboBox_id.setItemText(4, _translate("setting_camera", "4"))
        self.comboBox_id.setItemText(5, _translate("setting_camera", "5"))
        self.comboBox_id.setItemText(6, _translate("setting_camera", "6"))
        self.comboBox_id.setItemText(7, _translate("setting_camera", "7"))
        self.comboBox_id.setItemText(8, _translate("setting_camera", "8"))
        self.comboBox_id.setItemText(9, _translate("setting_camera", "9"))
        self.comboBox_conn.setItemText(0, _translate("setting_camera", "GigE"))
        self.comboBox_conn.setItemText(1, _translate("setting_camera", "USB 3"))
        self.comboBox_Brand.setItemText(0, _translate("setting_camera", "Basler"))
        self.comboBox_Brand.setItemText(1, _translate("setting_camera", "CSR"))
        self.comboBox_Brand.setItemText(2, _translate("setting_camera", "WC + DSHOW"))
        self.comboBox_Brand.setItemText(3, _translate("setting_camera", "WC + Default"))
        self.comboBox_Brand.setItemText(4, _translate("setting_camera", "Point Grey"))
        self.comboBox_Brand.setItemText(5, _translate("setting_camera", "IP Camera"))
        self.comboBox_Brand.setItemText(6, _translate("setting_camera", "Hik Vision"))
        self.comboBox_Brand.setItemText(7, _translate("setting_camera", "Imaging source"))
        self.comboBox_Flip.setItemText(0, _translate("setting_camera", "None"))
        self.comboBox_Flip.setItemText(1, _translate("setting_camera", "Vertically"))
        self.comboBox_Flip.setItemText(2, _translate("setting_camera", "Horizontally"))
        self.comboBox_Flip.setItemText(3, _translate("setting_camera", "Both"))
        self.comboBox_Rotate.setItemText(0, _translate("setting_camera", "0"))
        self.comboBox_Rotate.setItemText(1, _translate("setting_camera", "90ClockWise"))
        self.comboBox_Rotate.setItemText(2, _translate("setting_camera", "90CounterClockWise"))
        self.comboBox_Rotate.setItemText(3, _translate("setting_camera", "180"))
        self.pushButton_ok.setText(_translate("setting_camera", "OK"))
        self.pushButton_thoat.setText(_translate("setting_camera", "Tho√°t"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    setting_camera = QtWidgets.QWidget()
    ui = Ui_setting_camera()
    ui.setupUi(setting_camera)
    setting_camera.show()
    sys.exit(app.exec_())
